Annotato/Annotato/Persistence/DocumentsPersistenceManager.swift

```diff
@@ -103,12 +133,17 @@ extension DocumentsPersistenceManager {
         let senderId = decodedMessage.senderId
         let messageSubtype = decodedMessage.subtype
 
-        guard senderId != AuthViewModel().currentUser?.id else {
-            return
+        Task {
```

Comment:
We do not need to wrap this after changing the method to non-async.

My bad I forgot to remove this while refactoring.